docs/api-docs/getting-started/rest-api-authentication.md

```diff
@@ -174,9 +175,12 @@ All OAuth scopes except `default` provide `read-only` permissions scopes so that
 | Channel Settings | read-only | `store_channel_settings_read_only` | View a list of channels | [/v3/channels](/api-reference/cart-checkout/channels-listings-api) |
 | Channel Listings | modify | `store_channel_listings` | View and modify a list of all channel listings for a particular channel | [/v3/channels/listings](/api-reference/cart-checkout/channels-listings-api) |
 | Channel Listings | read-only | `store_channel_listings_read_only` | View a list of all channel listings for a particular channel | [/v3/channels/listings](/api-reference/cart-checkout/channels-listings-api) |
+|  | read-only |  | View web analytics | [](/...) |
```

Comment:
Need to confirm that the GraphQL DataLayer API authenticates using Store API Accounts (like the management APIs that are REST APIs)

Reply:
See comment [here](https://github.com/bigcommerce/api-specs/pull/1137/files#r1089433930)